app/scripts/modules/core/src/task/verification/UserVerification.tsx

```diff
@@ -45,7 +46,8 @@ export function UserVerification(props: IUserVerificationProps) {
       <div className="col-sm-12">
         <div className="form-inline">
           <div className="form-group">
-            <div className="form-control-static">{label || defaultLabel}</div> <HelpField id="user.verification" />
+            <div className="form-control-static">{label ? <Markdown tag="span" message={label} /> : defaultLabel}</div>{' '}
```

Comment:
is this `{' '}` necessary, or was it injected by prettier?

Reply:
prettier